### DISCUSSION TITLE
Can I do this in vendure

### DISCUSSION THREAD

**Discussion by emeczku** [vendurehq/vendure#2560] [General]
Hi, I wanted to ask if I can customize the backend and admin panel so that a user can create a new sale as an inquiry to the store, and the store will mark this sale as accepted for processing or not.

**Comment by michaelbromley** [comment]
Hi,

Yes, you can read more about customization of the admin panel here: https://docs.vendure.io/guides/extending-the-admin-ui/getting-started/

To implement an enquiry/acceptance flow, you would define a custom OrderProcess, as covered here: https://docs.vendure.io/guides/core-concepts/orders/#custom-order-processes
